lib/connect-backbone-to-react.js

```diff
@@ -105,6 +105,10 @@ module.exports = function connectBackboneToReact(
         this.setState(mapModelsToProps(this.models));
       }
 
+      componentWillReceiveProps(nextProps) {
```

Comment:
ah, good call. `componentWillReceiveProps` now calls `createNewProps`.

@hswolff , i've broken out `setModels`, which validates and sets `this.models`.

it makes sense to me that `props` would always take priority over `context`, no matter the order, since `props` is a lower-level specification. i've updated `componentWillReceiveProps` to behave this way.